lib/TraackrApi/Influencers.php

```diff
@@ -437,6 +437,62 @@ public static function search($p = array(
         return $inf->post(TraackrApi::$apiBaseUrl . 'influencers/search', $p);
     }
 
+    /**
+     * Stream influencers.
+     *
+     * @param array $p
+     * @return bool|mixed
+     * @throws MissingParameterException
+     */
+    public static function stream($p = array(
+        'is_tag_prefix' => false,
+        'gender' => 'all',
+        'type' => 'person',
+        'lang' => 'all',
+        'enable_tags_aggregation' => false,
+        'enable_country_aggregation' => false,
+        'enable_audience_aggregation' => false,
+        'enable_uids_aggregation' => false,
+        'batch_size' => 25,
+        'sort' => 'name',
+        'sort_order' => 'asc'
```

Comment:
[suggestion] We may want to include a `count` too. When I tried using the search-api endpoint in postman without a `count` it didn't seem like a safe default value was set (I got an error `too many influencers to stream`)